families/block_info/sawtooth_block_info/src/state.rs

```diff
@@ -211,6 +212,18 @@ impl<'a> From<&'a protos::block_info::BlockInfo> for BlockInfo {
     }
 }
 
+impl From<protos::block_info::BlockInfo> for BlockInfo {
```

Comment:
Why does this change need to be added?

Reply:
To support the modification in the diff chunk preceding this one, which previously used a reference but does not after this change.

Reply:
Should this be in a separate commit?

Reply:
Not sure if that's even possible or makes sense, given it is directly required/attached to the version change of protobuf. (Also, should be non-functional change.)